android/app/src/main/AndroidManifest.xml

```diff
@@ -46,7 +46,7 @@
                 <category android:name="android.intent.category.DEFAULT"/>
                 <category android:name="android.intent.category.BROWSABLE"/>
                 <data
-                    android:host="earth.hypha"
+                    android:host="hyphawallet.page.link"
```

Comment:
im not sure we need this change 

https://hypha.earth/invite?code=a6c3d5da41700cbe&chain=telos&dao=219987&env=dev

the deeplink is `https://hypha.earth`

Reply:
Bit unclear on this - maybe we need both, not sure